docs/tutorials/getting_started.rst

```diff
@@ -86,601 +117,85 @@ instance, for Mark 5B files one needs the number of channels used, as well as
     samples_per_frame = 5000
     sample_shape = (8,)
 
-The information is gleaned from ``info`` properties on the various file and
-stream readers (see below).
-
-.. note:: The one format for which ``file_info`` works a bit differently is
-   `GSB <gsb>`_, as this format requires separate time-stamp and raw data
-   files. Only the timestamp file can be inspected usefully.
+This information is extracted by `baseband.file_info`, which can also help
```

Comment:
Not necessary here. Maybe mention it with `open` above.

Reply:
Moved it to the "`baseband.open` tries to open files..." paragraph, shortened slightly.